src/mcp/server/__init__.py

```diff
@@ -114,6 +114,7 @@ def __init__(self, name: str):
 
     def create_initialization_options(
         self,
+        version: str | None = None,
         notification_options: NotificationOptions | None = None,
         experimental_capabilities: dict[str, dict[str, Any]] | None = None,
```

Comment:
In that case, perhaps the server __init__ would be a better place to make the change

```
    def __init__(self, name: str):
```

since it could then be the last parameter?

```
    def __init__(self, name: str, version: str = None):
```

Reply:
That sounds good to me.

Reply:
I picked the first for now.